packages/esm-patient-banner-app/package.json

```diff
@@ -52,5 +52,7 @@
     "react-router-dom": "5.x",
     "rxjs": "6.x"
   },
-  "devDependencies": {}
+  "devDependencies": {
+    "webpack-stats-plugin": "1.0.3"
```

Comment:
This should be in the global `devDependencies`. That way only a single change is necessary.